fe/app/components/TeamElementRow/TeamElementRow.tsx

```diff
@@ -27,8 +27,8 @@ const TeamElementRow = ({ teamElements }: TeamElementRowProps) => {
         <div className="flex" key={i}>
           <Divider className="h-0.5 w-full bg-white mr-4 mt-10 md:mr-16 md:mt-[100px]" />
           <div className="flex gap-8 md:gap-16">
-            {group.map((teamElement) => (
-              <TeamElement key={i} {...teamElement} />
+            {group.map((teamElement, e) => (
```

Comment:
Perhaps a better name? Link `index` or something else eheh

Reply:
on it!